### PR TITLE
/applab and /gamelab updates

### PR DESCRIPTION
**Pull request by caleybrock**
- Responsive code.org/applab and code.org/gamelab
 - /applab is missing a teacher section, but this will be added in a separate PR

cc @poorvasingal and @tanyaparker 

AppLab after
image

AppLab before
image

GameLab after
image

GameLab before
image

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/tools.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#resources {
+  margin-top: 40px;
+}
+
+.tools-info {
+  border: 1px solid;
+  border-color: #0094ca;
```
Comment (caleybrock) [inline pegasus/sites.v3/code.org/public/css/tools.css:7]: I've never been able to figure out a way to import color variables here so for now they will match the other css files and just use the numbers.

**Comment by tanyaparker** [comment]
Can you include images for what they both look like on mobile too? Mostly for posterity. It lgtm.

**Comment by caleybrock** [comment]
screen shot 2017-11-03 at 5 55 40 pm

**Comment by caleybrock** [comment]
screen shot 2017-11-03 at 5 57 15 pm

**Pull request merged**